tests/validators/answers/test_number_answer_validator.py

```diff
@@ -175,3 +175,39 @@ def test_invalid_numeric_answers():
     ]
 
     assert validator.errors == expected_errors
+
+
+def test_invalid_maximum_minimum_value_from_answer_source():
+    answer = {
+        "id": "min-max-range",
+        "mandatory": False,
+        "minimum": {"value": {"identifier": "set-minimum", "source": "answers"}},
+        "maximum": {"value": {"identifier": "set-maximum", "source": "answers"}},
+        "type": "Number",
+    }
+
+    answers = {
+        "answers": [
+            answer,
+            {
+                "id": "set-minimum",
+                "type": "Number",
+                "minimum": {"value": -9_999_999_999_999_999},
+            },
+            {
+                "id": "set-maximum",
+                "type": "Number",
+                "maximum": {"value": 9_999_999_999_999_999},
+            },
+        ]
+    }
+
+    questionnaire_schema = QuestionnaireSchema(answers)
+
+    validator = NumberAnswerValidator(answer, questionnaire_schema)
+
+    validator.validate_value_in_limits()
+
+    errors = validator.errors
+    assert errors[0]["message"] == validator.MINIMUM_LESS_THAN_LIMIT
+    assert errors[1]["message"] == validator.MAXIMUM_GREATER_THAN_LIMIT
```

Comment:
We should test the full error output instead of just the message. i.e Like the others in this file.

Reply:
Updated to test the full error output 👍